vhdbuilder/packer/install-dependencies.sh

```diff
@@ -413,29 +413,20 @@ done
 # v1.16.13-hotfix.20200714.1
 # v1.17.7-hotfix.20200714.1
 # v1.17.9-hotfix.20200714.1
-# v1.18.4-hotfix.20200624.1
-# v1.18.6
+# v1.18.4-hotfix.20200626.1
+# v1.18.6-hotfix.20200723.1	
 K8S_VERSIONS="
-1.14.7-hotfix.20200408.1
-1.14.8-hotfix.20200529.1
-1.15.7-hotfix.20200326
-1.15.10-hotfix.20200408.1
-1.15.11-hotfix.20200714.1
-1.15.12-hotfix.20200714.1
-1.16.7-hotfix.20200601.1
-1.16.8.1
-1.16.9-hotfix.20200529.1
-1.16.10-hotfix.20200714.1
-1.16.13-hotfix.20200714.1
-1.17.3-hotfix.20200601.1
-1.17.4.1
-1.17.5.1
-1.17.7-hotfix.20200714.1
-1.17.9-hotfix.20200714.1
-1.18.1.1
-1.18.2-hotfix.20200624
-1.18.4-hotfix.20200624.1
-1.18.6
+v1.15.11-hotfix.20200714.1
+v1.15.12-hotfix.20200714.1
+v1.16.9-hotfix.20200529.1	
+v1.16.10-hotfix.20200714.1
+v1.16.13-hotfix.20200714.1
+v1.17.3-hotfix.20200601.1
```

Comment:
why keep three for 1.17?

Reply:
I guess 1.1.6 gets three too. not sure why 1.155 and 1.18 only get two.

Reply:
I am trying to keep 3 for each one but 1.15.10 is not in this table

https://dev.azure.com/msazure/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/53245/Hotfix-Releases?anchor=components-patched

Reply:
I also have 3 for 1.18

Reply:
having 3 is okay, anything other than the supported versions are kept for perf/stability. We just want to keep as many as wen can to optimize the perf.